recipe/meta.yaml

```diff
@@ -45,7 +55,6 @@ about:
   description: |
     grpc-google-iam-v1 is the IDL-derived library for the
     google-iam v1 service in the googleapis repository.
-
   doc_url: https://github.com/googleapis/googleapis
```

Comment:
Should the doc_url be https://cloud.google.com/python/docs/reference/grpc-iam/latest?

Reply:
```suggestion
  doc_url: https://cloud.google.com/python/docs/reference/grpc-iam/latest
```